dleq/go_dleq.go

```diff
@@ -0,0 +1,76 @@
+package dleq
+
+import (
+	csecp256k1 "github.com/athanorlabs/atomic-swap/crypto/secp256k1"
+
+	dleq "github.com/athanorlabs/go-dleq"
+	"github.com/athanorlabs/go-dleq/ed25519"
+	"github.com/athanorlabs/go-dleq/secp256k1"
+
+	dsecp256k1 "github.com/decred/dcrd/dcrec/secp256k1/v4"
+)
+
+// DefaultDLEq is the default DLEq prover.
+type DefaultDLEq = GoDLEq
```

Comment:
Should we add, in the comment, that there is only one implementation at the current time? Or is it still easy to point this at the Rust version?

Reply:
yeah I'll add that this is the only implementation at the moment